_observability-plugin/ppl/commands.md

```diff
@@ -256,6 +265,80 @@ search source=accounts | fields account_number, firstname, lastname | fields - a
 | Nanette | Bates
 | Dale    | Adams
 
+
+## parse
+
+Use the `parse` command to parse a text field using regular expression and append the result to the search result. 
+
+### Syntax
+
+```sql
+parse <field> <regular-expression>
+```
+
+Field | Description | Required
+:--- | :--- |:---
+field | A text field. | Yes
+regular-expression | The regular expression used to extract new fields from the given test field. If a new field name exists, it will replace the original field. | Yes
+
+The regular expression is used to match the whole text field of each document with Java regex engine. Each named capture group in the expression will become a new ``STRING`` field.
+
+*Example 1*: Create new field
+
+The example shows how to create new field `host` for each document. `host` will be the host name after `@` in `email` field. Parsing a null field will return an empty string.
```

Comment:
I think we prefer hostname as one word. Also, we should use present tense.

"This example demonstrates..."